<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1331</width>
    <height>977</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QSplitter" name="splitter_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QSplitter" name="splitter_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Project</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtLibSearch"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTreeWidget" name="treeLibs">
             <property name="selectionMode">
              <enum>QAbstractItemView::ExtendedSelection</enum>
             </property>
             <attribute name="headerVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QGroupBox" name="groupCats">
         <property name="title">
          <string>Categories</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>Name</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="txtCatSearch"/>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QTreeWidget" name="listCategories">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QGroupBox" name="groupDocs">
         <property name="title">
          <string>Documentation</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QTreeWidget" name="listDocumentation">
              <property name="selectionMode">
               <enum>QAbstractItemView::ExtendedSelection</enum>
              </property>
              <attribute name="headerVisible">
               <bool>false</bool>
              </attribute>
              <column>
               <property name="text">
                <string notr="true">1</string>
               </property>
              </column>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Cell</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtCellSearch"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listGroups">
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>View</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Name</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtViewSearch"/>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QListWidget" name="listViews"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>Messages</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QTextEdit" name="textMessages"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1331</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>New</string>
     </property>
     <addaction name="actionSession"/>
     <addaction name="separator"/>
     <addaction name="actionProject"/>
     <addaction name="actionGroup"/>
     <addaction name="actionUnion"/>
    </widget>
    <widget class="QMenu" name="menuRecent_Projects">
     <property name="title">
      <string>Recent Projects</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionRecent1"/>
     <addaction name="actionRecent2"/>
     <addaction name="actionRecent3"/>
     <addaction name="actionRecent4"/>
     <addaction name="actionRecent5"/>
    </widget>
    <addaction name="menuNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_As"/>
    <addaction name="separator"/>
    <addaction name="menuRecent_Projects"/>
    <addaction name="actionClear_Recent_File_Stack"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Settings</string>
    </property>
    <addaction name="actionProjects"/>
    <addaction name="actionTools"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionShow_Documents"/>
   <addaction name="actionShow_Categories"/>
  </widget>
  <action name="actionProject">
   <property name="text">
    <string>Project...</string>
   </property>
  </action>
  <action name="actionGroup">
   <property name="text">
    <string>Cell...</string>
   </property>
  </action>
  <action name="actionUnion">
   <property name="text">
    <string>View...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/exit</normaloff>:/exit</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionProjects">
   <property name="text">
    <string>Projects</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionTools">
   <property name="text">
    <string>Tools</string>
   </property>
  </action>
  <action name="actionRecent1">
   <property name="text">
    <string>actionRecent1</string>
   </property>
  </action>
  <action name="actionRecent2">
   <property name="text">
    <string>actionRecent2</string>
   </property>
  </action>
  <action name="actionRecent3">
   <property name="text">
    <string>actionRecent3</string>
   </property>
  </action>
  <action name="actionRecent4">
   <property name="text">
    <string>actionRecent4</string>
   </property>
  </action>
  <action name="actionRecent5">
   <property name="text">
    <string>actionRecent5</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/open</normaloff>:/open</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/save</normaloff>:/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As...</string>
   </property>
  </action>
  <action name="actionShow_Documents">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/documents</normaloff>:/documents</iconset>
   </property>
   <property name="text">
    <string>Show Documents</string>
   </property>
  </action>
  <action name="actionShow_Categories">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../icons.qrc">
     <normaloff>:/categories</normaloff>:/categories</iconset>
   </property>
   <property name="text">
    <string>Show Categories</string>
   </property>
  </action>
  <action name="actionCategory">
   <property name="text">
    <string>Category</string>
   </property>
  </action>
  <action name="actionSession">
   <property name="text">
    <string>Session...</string>
   </property>
  </action>
  <action name="actionClear_Recent_File_Stack">
   <property name="text">
    <string>Clear Recent File Stack</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
